nautobot.code-workspace

```diff
@@ -45,12 +87,8 @@
 				"request": "attach",
 				"connect": {
 					"host": "127.0.0.1",
-					"port": 6899
+					"port": "${input:nautobotport}"
```

Comment:
correct me if im wrong but should this be 
```suggestion
					"port": "${inputs:nautobotport}"
```

Reply:
https://code.visualstudio.com/docs/editor/variables-reference#_input-variables

This was working for me during testing, from the docs:

> This limitation is solved with input variables which have the syntax: ${input:variableID}. The variableID refers to entries in the inputs section of launch.json and tasks.json, where additional configuration attributes are specified. Nesting of input variables is not supported.